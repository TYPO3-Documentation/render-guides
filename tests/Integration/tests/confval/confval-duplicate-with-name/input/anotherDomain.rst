=========================
Confval in another domain
=========================

..  confval:: demo
    :name: another-demo
    :type: string
    :default: ``"Hello World"``
    :required: False

    Some Text
