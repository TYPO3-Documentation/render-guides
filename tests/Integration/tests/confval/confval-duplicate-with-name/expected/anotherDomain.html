<!-- content start -->
                <section class="section" id="confval-in-another-domain">
            <h1>Confval in another domain<a class="headerlink" href="#confval-in-another-domain" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline"><i class="fa-solid fa-paragraph"></i></a></h1>
            <section data-search-title="demo" data-search-id="confval-another-demo" data-search-facet="Option">
<dl class="confval">
    <dt id="confval-another-demo" class="d-flex justify-content-between">
        <div class="confval-header">
        <code class="sig-name descname"><span class="pre">demo</span></code>
                <a class="headerlink" href="#confval-another-demo" data-bs-toggle="modal" data-bs-target="#linkReferenceModal" data-id="confval-another-demo" data-rstCode=":confval:`demo &lt;somemanual:another-demo&gt;`"  title="Reference this configuration value"><i class="fa-solid fa-paragraph"></i></a>
        </div>
        <div class="confval-back-to-top">            <a href="#" class="backToTop" title="Back to top"><i class="fa-solid fa-arrow-up fa-xs"></i></a>        </div>
    </dt>
    <dd>
                <dl class="field-list simple">
                            <dt class="field-even">Type</dt>
                <dd class="field-even">string
                </dd>
                        <dt class="field-odd">Default</dt>
            <dd class="field-odd"><code>&quot;Hello World&quot;</code>
            </dd>
            </dl>
                <div class="confval-description">

    <p>Some Text</p>

        </div>
    </dd>
</dl>
</section>
    </section>
        <!-- content end -->
